src/components/shared/Form/form.scss

```diff
@@ -0,0 +1,131 @@
+.form-wrap {
+  height: 86vh;
+}
+
+.wrap-cont-login-signup {
+  display: flex;
+  margin-bottom: 30px;
+  height: 55px;
+  .signup, .login {
+    width: 50%;
+    text-align: center;
+    cursor: pointer;
+    margin: 0;
+    border-radius: 0;
+    font-weight: 400;
+  }
+  button.ui.button.login {
+    background: #0E71EB;
+    color: #fff;
```

Comment:
@Akinmyde nice job, however, it looks like our background color `background: #0E71EB`, is been repeated several times. Since it has been declared globally already in `index.scss`, it'll be D.R.Y to import and use the variable.